routes/ci.js

```diff
@@ -73,38 +74,58 @@ router.post('/webhook', function(req, res, next) {
   if (branch !== "gh-pages") {
     switch (event) {
       case "push":
-        repositoryModel.findOneRepository(
-          {
-            githubId: req.body.repository.owner.id,
-            name: req.body.repository.name
-          }
-        ).
-        then(function(result) {
-          var data = {
-            email: result.email,
-            gitUrl: req.body.repository.clone_url,
-            docTheme: "",
-          }
-          generator.executeScript({}, data, function(err, generatedData) {
-            if (err) {
-              console.log(err);
-            } else {
-              deploy.deployPages({}, {
-                email: result.email,
-                gitURL: req.body.repository.clone_url,
-                username: result.username,
-                uniqueId: generatedData.uniqueId,
-                encryptedToken: result.accessToken
-              })
+      let sha = req.body.head_commit.id;
+      let username = req.body.repository.full_name.split("/")[0];
+      let repoName = req.body.repository.name;
+      let formats = ['.md', '.rst'];
```

Comment:
what will happend when i include any external file, for example an image to the repository?